kruize-clowdapp.yaml

```diff
@@ -73,6 +73,10 @@ objects:
             value: "prometheus-k8s"
           - name: savetodb
             value: "true"
+          - name: local
+            value: "false"
+          - name: logAllHttpReqAndResp
+            value: "true"
```

Comment:
@khansaad "false" for `logAllHttpReqAndResp` makes sense here as well, we can override it on EE when required, also thoughts on switching to upper case?

Reply:
@saltgen We must debug the reasons for the failure of the `updateRecommendation` calls. This flag needs to be set to `true` until the issue is resolved.

Switched the variable to upper case as suggested.

Reply:
Ah okay, it seemed like `logAllHttpReqAndResp` was for kruize local.